package.json

```diff
@@ -12,6 +12,7 @@
     "@testing-library/react": "^12.1.3",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.26.0",
+    "clsx": "^1.1.1",
```

Comment:
Where do we use this library?

Reply:
https://www.npmjs.com/package/react-toastify
It says on their npm site that "clsx" is a dependancy, I thought it had to be installed, maybe I misunderstood the meaning

Reply:
Oh I see, basically that is the information for user, to know what's going to be installed with certain package. I am super glad that you checked that.